CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+
+## [[0.16.3-0.5.0](https://github.com/line/cosmwasm/compare/v0.14.0-0.4.0...0.16.3-0.5.0)] - 2022-03-02
+
+### Features
+
+* Add derive macro "IntoEvent" ([#161](https://github.com/line/cosmwasm/issues/161))
+* merge original version 0.16.3 ([#148](https://github.com/line/cosmwasm/issues/148))
+* pin dependent uint version 0.9.1
+* Merge branch 'cosmwasm-0-16-2' into versionup-work
+* support Uuid type and sha1_calculate API ([#145](https://github.com/line/cosmwasm/issues/145))
+* Add release automation config ([#108](https://github.com/line/cosmwasm/issues/108))
+
+### Fixes
```

Comment:
How about writing it based on PRs? As in the other versions below.

Reply:
This is generated by "devtools/update_changelog.sh"

Reply:
Oh... Well, then, please manage once manually.
we need formatting, but we haven't time..

Reply:
OK, updated.